From 769e33c3b7014375e5cc1979fd760a61e96e79a6 Mon Sep 17 00:00:00 2001
From: Roy Pledge <roy.pledge@nxp.com>
Date: Mon, 25 Apr 2016 14:42:29 -0400
Subject: [PATCH 390/666] fsl_iommu: Enable 4th PAMU for T208X SoCs

PME and DCE are connected to a 4th PAMU in T208X SoCs.  This
change configures that PAMU so that the devices can operate.

Signed-off-by: Roy Pledge <roy.pledge@nxp.com>
[Original patch taken from QorIQ-SDK-V2.0-20160527-yocto]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/powerpc/boot/dts/fsl/t2081si-post.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/powerpc/boot/dts/fsl/t2081si-post.dtsi b/arch/powerpc/boot/dts/fsl/t2081si-post.dtsi
index ae51bd2..d030f94 100644
--- a/arch/powerpc/boot/dts/fsl/t2081si-post.dtsi
+++ b/arch/powerpc/boot/dts/fsl/t2081si-post.dtsi
@@ -672,9 +672,9 @@
 
 	iommu@20000 {
 		compatible = "fsl,pamu-v1.0", "fsl,pamu";
-		reg = <0x20000 0x3000>;
+		reg = <0x20000 0x4000>;
 		fsl,portid-mapping = <0x8000>;
-		ranges = <0 0x20000 0x3000>;
+		ranges = <0 0x20000 0x4000>;
 		#address-cells = <1>;
 		#size-cells = <1>;
 		interrupts = <
-- 
2.7.4

